### PR TITLE
Homepage & Explorepage 

### PR DESCRIPTION
**Pull request by varun-Tad**
Files to review

1. app folder
2. features folder
3. routes folder

Features to review

1. add to watch later
2. sidebar navigation
3. remove from watch later  (from explore page & watch later page)
4. watch later page navigation

Please review this PR at the earliest 
Preview: https://calm-smakager-a5972a.netlify.app/

### CODE REVIEW

File: dressvid/src/routes/WatchLaterpage/WatchLaterpage.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.main-section {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4rem;
+  margin-left: 18rem;
```
Comment (Omkar-Chaskar) [inline dressvid/src/routes/WatchLaterpage/WatchLaterpage.css]: Video section is not in center .  we can remove this line .
other than this everything looks good .

File: dressvid/src/App.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <div className="App">
+      <Sidebar />
+      <Routes>
+        <Route path="/" element={<Homepage />} />
+        <Route path="/explore" element={<Explorepage />} />
+      </Routes>
+    </div>
+  );
+}
+
```
Comment (Himanshu-Dhawale) [inline dressvid/src/App.js:22]: You can create a seperate file for route and then import it in App.js for clean coding purpose

**Review by Himanshu-Dhawale — APPROVED** [review]

**Review by iHarryD — COMMENTED** [review]
- Avoid using pixels as unit for larger values.
- Avoid using color names, instead use rgba or hex.
- Would recommend storing colors and margin, padding values in CSS variables for better consistency and alignment.
- You may remove the unused and commented codes.
- Use better naming. For example, you could rename showSidebar to toggleSidebar as it better provides the context.
- Avoid using indexes as keys. And be consistent with what you're using as keys.
- You could move all your routes to a common file and then put it in App.

**Review by iHarryD — APPROVED** [review]

**Comment by varun-Tad** [comment]
Thank you for the review.Will work on the suggestions

**Pull request merged**